Formula/suil.rb

```diff
@@ -29,12 +28,9 @@ class Suil < Formula
   depends_on "lv2"
   depends_on "qt@5"
 
-  # Disable qt5_in_gtk3 because it depends upon X11
-  # Can be removed if https://gitlab.com/lv2/suil/-/merge_requests/1 is merged
-  patch do
-    url "https://gitlab.com/lv2/suil/-/commit/33ea47e18ddc1eb384e75622c0e75164d351f2c0.diff"
-    sha256 "2f335107e26c503460965953f94410e458c5e8dd86a89ce039f65c4e3ae16ba7"
-  end
+  # Fix build issue, remove in next release
+  # upstream commit ref, https://github.com/lv2/suil/commit/7183178b8e35b9a05f2a90e1d091b34c5f846ef5
+  patch :DATA
```

Comment:
can't directly use the commit due to `patch: **** File waflib is not a regular file -- can't patch`